sn_client/src/docs/wallet/new.md

```diff
@@ -0,0 +1,10 @@
+# Arguments
+
+* `client.clone()` - A cloned instance of the struct: `sn_client::Client`
+* `wallet` - An instance of the struct `sn_transfers::LocalWallet`
+
+# Example
+
+ ```ignore
+ let mut wallet_client = WalletClient::new(client.clone(), wallet);
```

Comment:
`ignore` is not the same ` dont run`, I think ? (Are the docs compiled and verified?)

Here we're missing all the setup and related info that would make this useful, no? (the why as you said)